### ISSUE TITLE
Map layer flickers(blinks) as you pan map at zoom level 18

### ISSUE THREAD

**Issue by chimaobiv** [jupyter-widgets/ipyleaflet#507]
After tiling my geotif raster imagery, I tried to visualize it using ipyleaflet, i noticed above the zoom level 18, the raster layer disappears showing blank. And when you pan the map, the map starts to flicker, that is showing the raster layer off and on. The zoom layer for vector files does not produce this issue, only on tiled raster. How do I alter the zoom level to be above 18 for tiled raster geo tif images. Is this possible with ipyleaflet

**Comment by davidbrochart** [comment]
A Map has `max_zoom` and `min_zoom` attributes: https://ipyleaflet.readthedocs.io/en/latest/api_reference/map.html
You could set `max_zoom = 17` in your case.

**Comment by chimaobiv** [comment]
Yes @davidbrochart , I know I could set the min and max zoom. At 18, everything is good for me, but i need a higher zoom level to visualize certain features in the image. Is there a way I could bypass the max I. The Tileimage class?, or any other recommendations

**Comment by davidbrochart** [comment]
How do you serve your tiles, is your server publicly accessible (so that I can try)?
If you cannot share your setup, it would be nice to explain a bit more what you are doing and ideally have a minimal working example.
